test_files/enum/enum.js

```diff
@@ -93,8 +93,8 @@ EnumWithNonConstValues[EnumWithNonConstValues.Scheme] = 'Scheme';
 EnumWithNonConstValues[EnumWithNonConstValues.UserInfoRenamed] = 'UserInfoRenamed';
 /** @enum {string} */
 const StringEnum = {
-    STR: 'abc',
-    OTHER_STR: 'xyz',
+    STR: "abc",
```

Comment:
FWIW I typically merge change+test rebaseline in a single commit, so that tools like "git bisect" see a green test suite on every commit.  Pretty minor though.

Reply:
Good point, done.